api/src/main/java/com/debugmafia/clueless/state/Game.java

```diff
@@ -0,0 +1,144 @@
+package com.debugmafia.clueless.state;
+
+import com.debugmafia.clueless.actions.*;
+import com.debugmafia.clueless.model.*;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.*;
+
+
+import javax.tools.DocumentationTool.Location;
```

Comment:
Unused import

Reply:
I have VS Code for doing this to me

Reply:
Yea - it can be a pain for auto-imports at times I've noticed. For me it's also slick enough to let me know when I have unused stuff too though which is nice.